lib/records/P_RecCore.cc

```diff
@@ -862,11 +865,11 @@ reset_stat_record(RecRecord *rec)
 RecErrT
 RecResetStatRecord(const char *name)
 {
-  RecRecord *r1 = nullptr;
-  RecErrT err   = REC_ERR_FAIL;
+  RecErrT err = REC_ERR_FAIL;
 
-  if (ink_hash_table_lookup(g_records_ht, name, (void **)&r1)) {
-    err = reset_stat_record(r1);
+  auto it = g_records_ht.find(name);
```

Comment:
If you're willing to go full C++17, you could do
```
if (auto it = g_records_ht.find(name); it != g_records_ht.end()) {
  err = reset_stat_record(it->second);
}
```

Reply:
Yeah, this is nice.